recipes/libnuma/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.32.0"
```

Comment:
```suggestion
required_conan_version = ">=1.29.1"
```

for `tools.remove_files_by_mask()`